exec/cnex/global.c

```diff
@@ -174,6 +175,80 @@ TDispatch gfuncDispatch[] = {
     PDFUNC("os$system",                 os_system),
     PDFUNC("os$wait",                   os_wait),
 
+    // posix - posix OS functions
+    PDFUNC("posix$access",              posix_access),
+    PDFUNC("posix$chdir",               posix_chdir),
+    PDFUNC("posix$chmod",               posix_chmod),
+    PDFUNC("posix$chown",               posix_chown),
+    PDFUNC("posix$close",               posix_Close),
```

Comment:
Why is "Close" uppercased here? Is that a typo or does it conflict with something else called "posix_close"?

Reply:
Yes.  It conflicted, so the easy solution was to ust uppercase it in the code.

Reply:
What did it conflict with?

Reply:
I don't entirely know.  Looking back, I think it was an attempt to avoid any naming conflicts, but I don't see it as necessary now.